castle/cms/indexing/search.py

```diff
@@ -48,17 +54,25 @@ def __call__(self, hpscatalog, existing_data):
                 data[key + '_shares'] = value
         sdata = ISearch(self.obj, None)
         if sdata:
-            data['searchterm_pins'] = [
-                t.lower() for t in sdata.searchterm_pins or [] if t]
+            pins = sdata.searchterm_pins or []
```

Comment:
getattr to make it safe if property is ever not there?
`pins = getattr(sdata, 'searchterm_pins', [])`